addons/link_tracker/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,13 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_link_tracker_code(env):
```

Comment:
Doing this in post-migration, will provoke a warning on update phase if there's any record without code. For amending it, you can:

- In pre, fill existing records without code with a fixed string like `openupgrade_temp_code`.
- In post, do this, but searching for `('code', '=', 'openupgrade_temp_code')`